VoltronBaseApp/VRouterApi/src/main/java/com/voltron/router/api/VRouter.java

```diff
@@ -16,4 +16,12 @@ public static EndPointType resolveType(String path) {
     public static Postcard.Builder with(Context context) {
         return new Postcard.Builder(context);
     }
+
+    /**
+     * In general , clz is Activity or Fragment
```

Comment:
change clz to obj; clz is confusing.
/**
 * In general, obj is an instance of an Activity or a Fragment.
 */

Reply:
ok , i will change it later